lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -153,7 +154,9 @@ def cohort_students_and_upload(_xmodule_instance_args, _entry_id, course_id, tas
                 cohorts_status[cohort_name] = {
                     'Cohort Name': cohort_name,
                     'Students Added': 0,
-                    'Students Not Found': set()
+                    'Students Not Found': set(),
+                    'Invalid Email Addresses': set(),
+                    'Preregistered Users': 0
```

Comment:
@cahrens I stuck with "preregistered" partly because I couldn't think of anything better. But how about "pre-assigned"? @sstack22 ?